include/rocksdb/db.h

```diff
@@ -541,16 +537,20 @@ class DB {
     return Get(options, DefaultColumnFamily(), key, value, timestamp);
   }
 
-  // Returns all the merge operands corresponding to the key. If the
-  // number of merge operands in DB is greater than
-  // merge_operands_options.expected_max_number_of_operands
-  // no merge operands are returned and status is Incomplete. Merge operands
-  // returned are in the order of insertion.
-  // merge_operands- Points to an array of at-least
+  // Assigns all the merge operands corresponding to the key to the
+  // `merge_operands` array. Upon success, the number of merge operands set is
```

Comment:
Is "number of merge operands set" a typo?

Reply:
Rewrote to try to make it clearer.